api/urls.py

```diff
@@ -1,11 +1,12 @@
 from django.urls import path
 from rest_framework import routers
 from django.conf.urls import include
-from .views import UserViewSet, ProfileViewSet
+from .views import UserViewSet, ProfileViewSet,AddLoanViewSet
 
 router = routers.DefaultRouter()
 router.register('user', UserViewSet)
 router.register('profile', ProfileViewSet)
+router.register('loan/add', AddLoanViewSet)
```

Comment:
Did you test this route?

Reply:
yes I  did , that is the route for  creating loan